input/fsh/BFDR/bundlescompositions/CompositionJurisdictionFetalDeathReportNew.fsh

```diff
@@ -0,0 +1,272 @@
+  //   * ^short = "Plurality - Single, Twin, Triplet, etc."
+  //   * ^definition = "Plurality – The number of fetuses delivered live or dead at any time in the pregnancy regardless of gestational age or if the fetuses were delivered at different dates in the pregnancy. ('Reabsorbed' fetuses, those which are not 'delivered' (expulsed or extracted from the mother) should not be counted.)"
+* section[motherInformation] ^short = "Mother Administrative Section on the Fetal Death Report"
+  * ^definition = "This section contains items from the Mother administrative section on the Fetal Death Report."
+  * code 1.. 
+  * code = $loinc#92014-0
+    * ^short = "Mother's administrative information"
+    * ^definition = "Mother's administrative information"
+  * focus 1.. 
+  * focus only Reference(PatientMotherVitalRecords)
+    * ^short = "The focus of this section is the mother"
+    * ^definition = "The subject of the composition as a whole is the fetus baby. The focus of this section in the composition is the mother."
+  * entry 1.. 
+    * ^slicing.discriminator.type = #profile
+    * ^slicing.discriminator.path = "$this.resolve()"
+    * ^slicing.ordered = false
+    * ^slicing.rules = #open
+    * ^short = "Entries that are contained in the Mother Information section"
+    * ^definition = "Entries that are contained in the motherInformation section"
+  * entry contains mothersEducation 0..1 
+  * entry[mothersEducation] only Reference(ObservationEducationLevelVitalRecordsNew)
+    * ^short = "Mother's Education"
+    * ^definition = "Mother's education"
+* section[fatherInformation] ^short = "Mother Administrative Section on the Fetal Death Report"
+  * ^definition = "This section contains items from the Father administrative section on the Fetal Death Report."
+  * code 1.. 
+  * code = $loinc#92013-2
+    * ^short = "Father's administrative information"
+    * ^definition = "Father's administrative information"
+  * focus 1.. 
+  * focus only Reference(RelatedPersonFatherNaturalVitalRecords)
+    * ^short = "The focus of this section is the father"
+    * ^definition = "The subject of the composition as a whole is the fetus baby. The focus of this section in the composition is the father."
+* section[editFlags] ^short = "Edit Flags Section"
+  * ^definition = "This section contains the edit flags that specify whether the originally provided data passed validation checks. These validation checks are done at the jurisdiction prior to sending to the National Statistical Agency and are based on edit criteria in the Birth Edit Specifications for the 2003 Proposed Revision of the U.S. Standard Certificate of Birth and the Fetal Death Edit Specifications for the 2003 Revision of the U.S. Standard Report of Fetal Death."
+  * code 1.. 
+  * code = $loinc#92012-4
+    * ^short = "Edit flags section"
+  * entry ..8 
+    * ^slicing.discriminator.type = #profile
+    * ^slicing.discriminator.path = "$this.resolve()"
+    * ^slicing.rules = #open
```

Comment:
This should also include unordered (probably the default), so can use the to-be-found/defined ruleset